utilities/_layout.scss

```diff
@@ -0,0 +1,32 @@
+//Full responsive layout based on design system
+//Todo: amend spacing calc for between 414px and 599 as some misalignment
+.grid {
+    @include grid(repeat(6, 1fr), 16px, 24px);
```

Comment:
Should we have these gap values as variables within a `layouts` file in this directory? https://github.com/AgePartnership/oleg-starter-pack/tree/develop/variables @beccadawson thoughts?